components/SurveyModal.js

```diff
@@ -0,0 +1,566 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+// Callback function for user to complete mailchimp signup
+function newsletterCallback() {
+  const mailchimpForm = document.getElementById('mailchimp-form');
+  mailchimpForm.action =
+    'https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac';
```

Comment:
Should we stick our Mailchimp URL in a shared variable somewhere?

Reply:
I think this is a good idea, but I'm not sure how we should go about this.

The two options that came to mind for me are to either use React's Context API, or Netlify's environment variables. Any thoughts?

Reply:
Both of those options make sense to me. I think if this key were sensitive (i.e. a malicious user could find it on GitHub and do something bad with it) or if we wanted to use different keys in production and development, environment variables would be a clear winner. Since neither of those things are true right now, I'd be down with whichever approach you find easier to set up and maintain.

I'd be cool with merging this PR as-is and opening up a separate issue and PR to move this URL into a shared variable somewhere.